hw/ip/otbn/util/otbn-objdump

```diff
@@ -146,6 +146,7 @@ def main() -> int:
             proc = subprocess.run(cmd, capture_output=True, text=True)
             if proc.returncode:
                 # Dump any lines that objdump wrote before it died
+                sys.stderr.write(proc.stderr)
                 sys.stdout.write(proc.stdout)
```

Comment:
Optional: I'd probably to for print() here to ensure that there's a newline between the prints. And potentially also add a print("STDERR:") or something like that, since the outputs are not multiplexed (and there's no easy way to mux them at this point).

Reply:
I think I'll probably not add the STDERR prefix: these are coming out on different FDs (just as they would have done if the OTBN wrapper wasn't there at all). Similarly with adding newlines: I'd rather just pass whatever came from `objdump` (in practice, this is newline terminated anyway).

Let me know if (on reflection!) you think this is completely wrong and I'll file another PR to fix it up.